docs/topics/settings.rst

```diff
@@ -218,6 +218,26 @@ Default: ``None``
 
 The name of the region associated with the AWS client.
 
+.. setting:: ASYNCIO_EVENT_LOOP
+
+ASYNCIO_EVENT_LOOP
+------------------
+
+Default: ``None``
+
+Import path of a given asyncio event loop.
+
+If the asyncio reactor is enabled this setting can be used to specify the 
+asyncio event loop to be used with it. Set the setting to the import path of the 
+desired asyncio event loop class. If the setting is set to ``None`` the default asyncio
+event loop will be used.
+
+You can also specify the event loop path with the function :func:`~scrapy.utils.reactor.install_reactor`:
```

Comment:
It sounds like, as an alternative to using the `ASYNCIO_EVENT_LOOP` setting, you can use a parameter of the `install_reactor` function.

Instead, what it should say is that, if you are installing the asyncio reactor manually, the `ASYNCIO_EVENT_LOOP` setting has no effect. If you are installing it with `install_reactor`, you can use the `event_loop_path` parameter to indicate the import path of the event loop to use.